### PR TITLE
AM-6265: Update the gateway-sidecar README.md file

### PR DESCRIPTION
**Pull request by uma-kt**
Signed-off-by: uma-kt <105046156+uma-kt@users.noreply.github.com>

- Rephrased
- Fixed typos
- Fixed hyperlinks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-
-* Docker installed and running in your local machine
-* A running [`kind`](https://kind.sigs.k8s.io/)
-* [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed and configured
-* Follow the getting started from above, to install [`kubeslice-controller`](https://github.com/kubeslice/kubeslice-controller) and [`worker-operator`](https://github.com/kubeslice/worker-operator)
+Before you begin, make sure the following prerequisites are met:
+* Docker is installed and running on your local machine.
+* A running [`kind`](https://kind.sigs.k8s.io/).
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/) is installed and configured.
+* You have prepared the environment for the installation of [`kubeslice-controller`](https://github.com/kubeslice/kubeslice-controller) on the controller cluster
```
Comment (veenamj-avesha) [inline README.md:17]: Use for installing or to install

File: README.md [hunk 2]
```diff
@@ -46,7 +47,7 @@ git clone https://github.com/kubeslice/gateway-sidecar.git
 cd gateway-sidecar
 ```
 
-2. Adjust `VERSION` variable in the Makefile to change the docker tag to be built.
+2. Adjust the `VERSION` variable in the Makefile to change the docker tag to be built.
 Image is set as `docker.io/aveshasystems/kubeslice-gw-sidecar:$(VERSION)` in the Makefile. Change this if required
```
Comment (veenamj-avesha) [inline README.md:51]: Change this .. should end with a period.

File: README.md [hunk 3]
```diff
@@ -80,7 +81,7 @@ docker exec -it kind-control-plane crictl images
 
 ### Deploy in a Cluster
 
-Update chart values file `yourvaluesfile.yaml` that you have previously created.
+Update the chart values file `yourvaluesfile.yaml` that you have previously created.
```
Comment (veenamj-avesha) [inline README.md:84]: add a comma after values file

**Review by veenamj-avesha — COMMENTED** [review]
Left my feedback.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
-# gateway-sidecar
+# Gateway Sidecar
 
 The Slice VPN Gateway is a slice network service component that provides a secure VPN tunnel between any two clusters that are a part of the slice.
```
Comment (veenamj-avesha) [inline README.md:3]: As per Eric's suggestion, add more info on the purpose. 

Something like - Installing gateway-sidecar provides a Slice VPN Gateway, a slice network... 

or check with the dev team for the purpose. 

cc @rajendra-avesha / @gourishkb

File: README.md [hunk 5]
```diff
@@ -1,20 +1,21 @@
-# gateway-sidecar
+# Gateway Sidecar
```
[the remark this answers is not in the document]
Comment (uma-kt) [reply README.md:1]: @veenamj-avesha I did check this with @narmidm.  He is okay with this convention for documentation.
I have referred to other docs in google, some docs have repo/artifact names, and in some conventions are used.

File: README.md (hunk 4 above)
Comment (uma-kt) [reply README.md:3]: @veenamj-avesha  This PR was raised to fix the hyperlinks. I will work on the introduction in a separate PR. As dev team is busy with the release now

**Pull request merged**